src/ImageSharp.Drawing/Processing/Processors/Text/DrawTextProcessor{TPixel}.cs

```diff
@@ -90,26 +90,31 @@ void Draw(List<DrawingOperation> operations, IBrush brush)
                             Buffer2D<float> buffer = operation.Map;
                             int startY = operation.Location.Y;
                             int startX = operation.Location.X;
-                            int offSetSpan = 0;
+                            int offsetSpan = 0;
                             if (startX < 0)
                             {
-                                offSetSpan = -startX;
+                                offsetSpan = -startX;
                                 startX = 0;
                             }
 
-                            int fistRow = 0;
+                            if (startX >= source.Width)
+                            {
+                                continue;
```

Comment:
I'm not sure if `continue` here is actually the best idea and not just short-circuiting with a `break`. I might look into this further just so the code doesn't need to loop additionally for no reason.

Reply:
After a bit more looking into this, it seems like it could break (no pun intended) certain brushes depending what direction they were drawing (eg. backward diagonal) if I were to `break` instead of `continue`. I could write a short circuit for it depending on the type of brush but that feels a little overkill.